code/modules/admin/verbs/beakerpanel.dm

```diff
@@ -65,8 +65,11 @@
 	set name = "Spawn reagent container"
 	if(!check_rights())
 		return
+/*
+	// Maybe some day, but this doesn't work without a client
 	var/datum/asset/asset_datum = get_asset_datum(/datum/asset/simple/namespaced/common)
 	asset_datum.send()
+*/
```

Comment:
src is the client in admin datums.

don't ask.

ok fine, because its a verb that gets added to the client, and verbs always have their attached datum as the src, and here that's the client, even thou src is typed to the admin datum, it will contain the client.

Reply:
The problem I have is it runtimes saying the client does not exist for some reason and I am not sure why it never gets attached?  I marked it out but common.css was apparently loaded by some other asset..

..We really need to mmake some kind of mega common css for these things

Reply:
i mean, don't look at client.

look at `src` or `src||usr?.client`

Reply:
is that documented anywhere?